### DISCUSSION TITLE
Question: how to specify subgraph-global node properties

### DISCUSSION THREAD

**Discussion by svdvonde** [ts-graphviz/ts-graphviz#800] [Q&A] [answered]
Dear maintainers of ts-graphviz,

First of all thank you for this library! I am wondering how I can use the library to generate graphs such as: 

```
digraph G {
    node [style=filled, border=1];
    a0;
}
```

I cannot seem to figure out how to add a property such as `node [style=filled, border=1];` that applies a style to all nodes in the subgraph. They do not seem to be regular properties.

**Comment by kamiazya** [comment, marked as answer]
@svdvonde 

Thank you for using ts-graphviz.

You can pass properties to the Cluster object with the `attributes.node.apply` method.


Please refer to the example below for running with runkit!

https://runkit.com/kamiazya/5fc68bf9a91e63001a093139

**Comment by svdvonde** [comment]
Thank you!

**Comment by kamiazya** [comment]
@all-contributors please add @svdvonde  for question.

[a comment by an automated account is not shown]
